dr_evaluation/app.py

```diff
@@ -0,0 +1,251 @@
+from sklearn.linear_model import RidgeCV
+from sklearn.metrics import mean_squared_error
+import pandas as pd
+import numpy as np
+import pymortar
+import logging 
+import os
+import json
+
+from get_test_days import get_test_data
+from feature_engineering import get_time_of_week, get_t_cutoff_values
+from utils import get_window_of_day, get_workdays, get_closest_station
+from daily_data import get_daily_data
+import get_data as gd
+
+mpl_logger = logging.getLogger('matplotlib') 
+mpl_logger.setLevel(logging.WARNING) 
+
+def fit(site, start_train, end_train, cli, exclude_dates=[]):
+    """
+    Fit the regression model for a site during for the specified window
+    exclude_dates is a an optional set of datetime.date objects to exclude from training
+    cli: pymortar client
+    """
+    start = start_train
+    end = end_train
+    interval = '15m'
+    agg = 'MEAN'
+    alphas = [0.0001, .001, 0.01, 0.05, 0.1, 0.5, 1, 10]
+
+    # Get weather
+    weather = gd.get_weather(site, start, end, agg=agg, window=interval, cli=cli)
+    weather.index = pd.date_range(start, end, freq='15min')[:-1]
```

Comment:
This should probably refer to the `interval variable`

Reply:
done